### ISSUE TITLE
rocks breaks for tables with a single entry

### ISSUE THREAD

**Issue by bcarry** [maxmahlke/rocks#1]
If rocks is used to query a datacloud table with a single entry, the class bugs

from rocks import rocks
scheila = rocks(596, datacloud='masses')[0]
scheila.masses

masses contains a single entry for 596
The same code works smoothly for, e.g., 1, 2, 3 4....

**Comment by maxmahlke** [comment]
Could you include the error message? On my side it does not throw an error, thought the representation on the CLI shows lists instead of strings,

```
➜ ~ rocks masses Scheila        
┌────────────┬────────────┬─────────────┬─────────────────┬────────┐
│ mass       │ err_mass   │ method      │ shortbib        │ year   │
├────────────┼────────────┼─────────────┼─────────────────┼────────┤
│ [5.37e+18] │ [1.19e+18] │ ['DEFLECT'] │ ['Goffin+2014'] │ [2014] │
└────────────┴────────────┴─────────────┴─────────────────┴────────┘
                           (596) Scheila
```

**Comment by maxmahlke** [comment]
Should be fixed with  08b4746, could you check?

**Comment by bcarry** [comment]
Yes. Issue solved
